### PR TITLE
Add timeout feature

### LINKED ISSUE #6: Add timeout feature
I would like to implement the feature of checking for timeouts while making network requests.

### PR DESCRIPTION
**Pull request by mshinh**
#### Fixed #6 

- Added Timeout of 1.5 secs to axios (Optimization)

- Updated README.md to include this feature and elaborated on the existing features

- Improved styling and indentation

### CODE REVIEW

**Comment by chawlapalak** [comment]
Hi Muskan:

I am unable to see your branch in my repository. Also, there is a merge conflict in the Readme file. So please resolve this merge conflict and add the changes in the branch visible to me. 

Also, the branch name should be the issue number.

Thanks,
Palak

**Comment by mshinh** [comment]
Hey Palak, 
You cannot see the branch in your repo because I forked your repo, cloned the one I forked, made a branch in that repo and then made changes and pushed. 
So I created this pull request from the branch in my forked repo to your master branch. This is what I think we have to do. 
Also, the readme file conflict will be resolved once you review the file and accept the changes, I think. 

I made the mistake of branch name but I don't think I can change that now.

**Comment by mshinh** [comment]
Hey Palak, 
I resolved the conflict in the readme file. There are no more conflicts.

Thanks,
Muskan

File: README.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
 # URL-Inspector
 This is a command line interface tool to check different URLs and detect if they are good or bad or unknown.
 
-# Installation
+# Instalation
```
Comment (chawlapalak) [inline README.md]: Spelling mistake, please correct.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+* Version
+    url-inspector -v or -V or --version
+
 # Features
+* It displays URLS in the following colors
+    * Good or Status 200 is displayed in green
+    * Bad or Status 400 or 404 is displayed in red
+    * Unknown status is displayed in gray
+* It displays the toolname along with its current version with the following command
+    url-inspector -v or -V or --versin
```
Comment (chawlapalak) [inline README.md]: Version spelled wrong. Please fix.

**Review by chawlapalak — CHANGES_REQUESTED** [review]
Hi, 
Please correct the spellings in the readme file. Thanks.

Best,
Palak

File: README.md (hunk 1 above)
Comment (mshinh) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (mshinh) [reply README.md]: Fixed.

**Review by chawlapalak — APPROVED** [review]

**Comment by chawlapalak** [comment]
Hi,
Approved.
Thanks,
Palak Chawla

**Pull request merged**